vpr/src/place/place_util.cpp

```diff
@@ -492,7 +492,7 @@ bool macro_can_be_placed(t_pl_macro pl_macro, t_pl_loc head_pos, bool enable_ana
          * floorplan constraint is not supported by analytical placement yet, 
```

Comment:
This comment should also mention that you're not checking the capacity restriction since the analytic placer checks that separately.

The more I think about it the more I think this parameter should also be renamed to check_all_legality or some such and the comment should explain that the analytic placer doesn't check everything. A parameter called analytic_placer doesn"t really explain to a future user trying to figure out if they can call this or not what the value of this parameter should be (since the parameter name does not describe what the parameter controls)